### DISCUSSION TITLE
Moving Wiki under docs/

### DISCUSSION THREAD

**Discussion by snshn** [Lyrics/lyrics#494] [Ideas]
GitHub Wiki can't travel easily with the repo, and there's a lot of useful information there. I'm thinking it may be worth getting rid of it by creating .md files out of it under docs/. One of the downsides I see is that the CI/CD will fire up every time we modify those documents, but maybe there's a way to make it ignore modifications done to certain files and directories.

**Comment by andy5995** [comment]
>One of the downsides I see is that the CI/CD will fire up every time we modify those documents, but maybe there's a way to make it ignore modifications done to certain files and directories.

PR https://github.com/Lyrics/lyrics/pull/503 opened for this

**Comment by andy5995** [comment]
Hmm.. interesting... @sonny-mo just opened  https://github.com/Lyrics/lyrics/pull/504 and migrated the wiki to https://docs.contour.so/Lyrics/lyrics/manual-3w93hpzablu000000000

I was just editing the wiki docs in #503 to add links. I realized another downside @snshn ... navigation. Manually linking to each doc.. Easy enough the first time, but maintaining that... might be an "ugh!" situation. ;) But maybe you or someone will have some ideas about that. Here's an example in my last commit to #503 

https://github.com/Lyrics/lyrics/pull/503/commits/24f9421108920a4eb649b555a909dc27b6a4f359#diff-0474e525df6c10faef7f4f0dbee275b9f58d1312320b5a42d04be66850ec32a7
